deployments/audience_engagement.yaml

```
@@ -2,27 +2,32 @@ status: Converted
 registry_authors:
 - Nicolas Berrios
 - Jack Fitzsimons
+- James Honaker
 tier: 1
 deployment:
   name: Audience Engagement API
   data_curator: LinkedIn
   intended_use: Statistics API
   data_product_type: Summary statistics # TODO: Is this correct?
   data_product_region: Global
-  description: With these proposed parameter values (...) and applying
-    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee.
+  description: 'With these proposed parameter values (...) and applying
+    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee.'
```

If you want the quotation marks to be rendered downstream:
```suggestion
  description: '"With these proposed parameter values (...) and applying
    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee."'
```
Or since this is across a couple lines, block quotes style might be more clear:
```suggestion
  description: |
    "With these proposed parameter values (...) and applying
    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee."
```
Longer comment in slack.